docs/roadmaps/active_roadmap.md

```diff
@@ -0,0 +1,266 @@
+
+We welcome **input from the community** about needs and priorities. This document will be publicly posted and be easily discoverable from the napari website.
+
+### Potential challenges
+
+Common risks associated with roadmaps are:
+
+* **Scope Creep:** The influx of issues and pull requests can lead to scope creep, diverting focus from the core objectives.
+
+* **Lack of Clarity:** Ambiguous communication between the team and contributors can hinder progress and lead to misunderstandings.
+
+* **"How" Before "Why":** Prioritizing implementation details ("how") over the underlying rationale ("why") can cause misaligned efforts.
+
+* **Estimation Difficulties:** Accurately estimating project timelines and resource requirements is challenging.
+
+### Risk mitigation strategies
+
+These practices reduce risk and increase the likelihood of completing strategic deliverables:
+
+* **Transparent Roadmap:** Maintain a clear and easily discoverable roadmap to guide development and manage expectations.
+
+* **Regular Reviews:** Conduct quarterly reviews and updates to ensure the roadmap remains relevant and aligned with current priorities.
+
+* **Annual Deep Dives:** Perform a yearly in-depth analysis of each focus area to assess progress and identify areas for improvement.
+
+* **Iterative Development:** Embrace an iterative approach to development, incorporating lessons learned into future iterations.
+
+* **Visible Refactoring Plan:** Establish a clear plan for refactoring efforts and track progress beyond individual issues and pull requests.
+
+## Roadmap focus areas
+
+### Core technology
+
+* **API (Code interface):** The interface to the external scientific Python world, enabling extensibility, customization, and integration into pipelines and workflows.
+
+* **Library (Implementation):** The API implementation; new useful features across all interaction modes.
+
+* **Performance:** Optimization efforts to enhance speed and efficiency.
+
+* **Evolution:** Ongoing development and improvement of the core components.
+
+### User experience
+
+* **User Interfaces and Interaction Modes:** The bundled application (using Qt), new applications (NapariLite, browser, WebAssembly), notebook, and programmatic use cases.
+
+* **Plugin Ecosystem:** The engine for extensibility, input/output plugins, domain-specific functionality, and discoverability of plugins.
+
+### Project sustainability and community
+
+* **Bug Fixes:** Addressing reported issues and ensuring software stability.
+
+* **Documentation:** Maintaining comprehensive and up-to-date documentation.
+
+* **Website:** Providing a central hub for information and resources.
+
+* **Community Engagement:** Fostering communication, outreach, and collaboration through conferences and core team meetings.
```

Comment:
I think this is a strong point of napari and its community, and we could potentially highlight the approach to community engagement as part of the project's mission. One of the best things about a community-led project (and what give it an edge over proprietary systems, in general) is that engagement and a committment to listening to feedback from the community. I think napari does it well and we should highlight it.

Reply:
@DragaDoncila You may be able to use a `literalinclude` with line numbers using Sphinx/myst